### DISCUSSION TITLE
Ошибка при запуске сервера на reHLDS последней версии 3.13.0.788

### DISCUSSION THREAD

**Discussion by imgfilesbox** [rehlds/ReHLDS#1012] [Q&A]
Подскажите, пожалуйста, в чём может быть проблема.
Установил чистый сервер hl 1 через steamcmd, пославил на него rehlds 3.13.0.788, при запуске выдаёт такую ошибку. Без rehlds сервер работает исправно.
![image](https://github.com/dreamstalker/rehlds/assets/156510299/a1d28e79-f9b4-4106-9aab-ff09678ccac6)
![image](https://github.com/dreamstalker/rehlds/assets/156510299/27b574d7-131c-4137-94a9-8aa4166d9bc8)

**Comment by LoShIaA** [comment]
Hello, today I decided to update my platform, which I did back in 2021, and I got this error. 
I used the latest version rehlds-bin-3.14.0.857

Edit: I got this error on the latest platform that I downloaded from steamcmd (7 oct 2024 - 10210), now I tried on an older version and there is no problem.


Is there any solution??

**Comment by Nord1cWarr1or** [comment]
Устанавливать нужно на ветку `steam_legacy` из steamcmd
